packages/express-react/src/server/render.tsx

```diff
@@ -4,7 +4,7 @@ import { Writable } from 'stream'
 import { type ReactNode } from 'react'
 
 import {
-  createRootStyleCache,
```

Comment:
Here and below, replace server-side `createRootStyleCache` calls w/ `createFlushableStyleCache`.